check_process

```diff
@@ -12,7 +12,7 @@
 		setup_private=0
 		setup_public=0
 		upgrade=1
-		upgrade=1	from_commit=9c57c2f78f3da979d587d708eb9379ffaa41bcfa
+		upgrade=1	from_commit=af36640ee5a5f1fab69651d638d23ebe47198152
```

Comment:
Why did you change this commit?

Reply:
The Ci fails when upgrading from this commit and the test is ignored... My idea was to test the upgrade from 18.0.7 to 19.0.3 but I can revert back to previous state if you think its better

Reply:
Hmm, the problem is that php-mcrypt does not exist. This commit should be the oldest one that works: (not tested)
https://github.com/YunoHost-Apps/nextcloud_ynh/pull/253/commits/c864e086625363d91dde6ba78a652011a991a027

Reply:
c864e08 tested and OK